Mathlib/Topology/UrysohnsLemma.lean

```diff
@@ -88,50 +88,55 @@ open scoped Pointwise
 namespace Urysohns
 
 
-/-- An auxiliary type for the proof of Urysohn's lemma: a pair of a closed set `C` and its
-open neighborhood `U`, together with the assumption that `C` satisfies the property `P C`. The
-latter assumption will make it possible to prove simultaneously both versions of Urysohn's lemma,
-in normal spaces (with `P` always true) and in locally compact spaces (with `P = IsCompact`).
-We put also in the structure the assumption that, for any such pair, one may find an intermediate
-pair in between satisfying `P`, to avoid carrying it around in the argument. -/
-structure CU {X : Type*} [TopologicalSpace X] (P : Set X → Prop) where
+/--
+An auxiliary type for the proof of Urysohn's lemma: a pair of a closed set `C` and its open
+neighborhood `U`, together with the assumption that `C` and `U` satisfy the property `P C U`.
+The latter assumption will make it possible to prove simultaneously both versions of Urysohn's
+lemma, in normal spaces (with `P` always true) and in locally compact spaces
+(with `P C U = IsCompact C`). We put also in the structure the assumption that, for any such pair,
+one may find an intermediate pair in between satisfying `P`,
+to avoid carrying it around in the argument.
+-/
```

Comment:
Changing the comment formatting style seems orthogonal to the content of this PR. (I personally prefer the old style, BTW.)